shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/uicases/trades/TradeFlowPresenter.kt

```diff
@@ -0,0 +1,68 @@
+package network.bisq.mobile.presentation.ui.uicases.trades
+
+import androidx.compose.runtime.Composable
+import cafe.adriel.lyricist.LocalStrings
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.MutableStateFlow
+import network.bisq.mobile.domain.data.model.OfferListItem
+import network.bisq.mobile.domain.service.offerbook.OfferbookServiceFacade
+import network.bisq.mobile.presentation.BasePresenter
+import network.bisq.mobile.presentation.MainPresenter
+
+enum class TradeFlowScreenSteps(val titleKey: String) {
+    ACCOUNT_DETAILS(titleKey = "bisqEasy_tradeState_phase1"),
+    FIAT_PAYMENT(titleKey = "bisqEasy_tradeState_phase2"),
+    BITCOIN_TRANSFER(titleKey = "bisqEasy_tradeState_phase3"),
+    TRADE_COMPLETED(titleKey = "bisqEasy_tradeState_phase4")
+}
+
+@Composable
+fun TradeFlowScreenSteps.getTranslatedTitle(): String {
+    val strings = LocalStrings.current.bisqEasyTradeState
+    return when (this) {
+        TradeFlowScreenSteps.ACCOUNT_DETAILS -> strings.bisqEasy_tradeState_phase1
+        TradeFlowScreenSteps.FIAT_PAYMENT -> strings.bisqEasy_tradeState_phase2
+        TradeFlowScreenSteps.BITCOIN_TRANSFER -> strings.bisqEasy_tradeState_phase3
+        TradeFlowScreenSteps.TRADE_COMPLETED -> strings.bisqEasy_tradeState_phase4
+    }
+}
+
+// TODO: Should do Interface for this?
```

Comment:
probably yes since this handles the wholw trade flow. This presenter is very important, it will be interacting with services and/or repositories to manage the real world data and views should be consuming what they need through the interface that this presenter will implement (see other existing examples but I'm sure you know what I'm talking about already :) )

Reply:
Yeah. Got it